apps/docs/scripts/troubleshooting/watch.mjs

```diff
@@ -0,0 +1,68 @@
+// @ts-check
+
+import chokidar from 'chokidar'
+import { closeSync, openSync, readFileSync, writeSync } from 'node:fs'
+import { basename, dirname, join } from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = dirname(__filename)
+const TROUBLESHOOTING_DIRECTORY = join(__dirname, '../../content/troubleshooting')
+
+let __template = ''
+
+function getTemplate() {
+  if (!__template) {
+    __template = readFileSync(join(TROUBLESHOOTING_DIRECTORY, '_template.mdx'), 'utf8')
+  }
+  return __template
+}
+
+const watcher = chokidar
+  .watch(TROUBLESHOOTING_DIRECTORY, { ignoreInitial: true })
+  .on('add', (path, stats) => {
+    if (!stats?.isFile() || basename(path).startsWith('_')) return
+
+    console.log(`File ${path} has been added.`)
+
+    if (stats?.size !== 0) {
+      console.log(`File ${path} already has content. Skipping template writing.`)
+      return
+    }
+
+    try {
+      console.log(`Writing template to ${path}...`)
+      const fd = openSync(path, 'w')
+      writeSync(fd, getTemplate())
+      closeSync(fd)
+    } catch (err) {
+      console.error(`Error writing template to ${path}:`, err)
+    }
+  })
+
+// Keep the process alive
+const interval = setInterval(() => {}, 1000)
```

Comment:
Strange you need this since chokidar's [persistence flag](https://github.com/paulmillr/chokidar?tab=readme-ov-file#persistence) should have handled this.

Reply:
Yeah I thought it would automatically work too, no idea what went wrong